### PR TITLE
Change default pictures to identicons.

### PR DESCRIPTION
**Pull request by hibooboo2**

### CODE REVIEW

File: code/framework/api/src/main/java/io/cattle/platform/api/auth/Identity.java [hunk 1]
```diff
@@ -8,9 +9,12 @@
 import org.apache.commons.lang3.builder.ToStringBuilder;
 import org.springframework.util.DigestUtils;
```
Comment (ibuildthecloud) [inline code/framework/api/src/main/java/io/cattle/platform/api/auth/Identity.java]: ah.... don't import spring.  don't need to change, but don't import spring classes in the future.

Comment (hibooboo2) [inline code/framework/api/src/main/java/io/cattle/platform/api/auth/Identity.java]: How do i use DigestUtils then?

Comment (ibuildthecloud) [inline code/framework/api/src/main/java/io/cattle/platform/api/auth/Identity.java]: You don't.  Find a different approach.  We will eventually drop Spring.

Comment (kaos) [inline code/framework/api/src/main/java/io/cattle/platform/api/auth/Identity.java]: :stuck_out_tongue_closed_eyes:

**Pull request merged**